data/scripts/movements/snow.lua

```diff
@@ -2,7 +2,11 @@ local snow = MoveEvent()
 
 function snow.onStepOut(creature, item, position, fromPosition)
 	local player = creature:getPlayer()
-	if player and player:isInGhostMode() then
+	if not player then
+		return true
+	end
+
+	if player:isInGhostMode() then
```

Comment:
```suggestion
	if not player or player:isInGhostMode() then
```

Reply:
@dudantas In the case of OR, if it is not a player, it will check the second condition, and then it will give an error!

Reply:
> @dudantas In the case of OR, if it is not a player, it will check the second condition, and then it will give an error!

True.